doc/scalar/riscv-crypto-scalar-entropy-source.adoc

```diff
@@ -47,322 +56,285 @@ machine-mode CSR described in the table below.
 |=======================================================================
 
 The instruction is *non-blocking* and returns immediately, either with
-two status bits `mentropy[31:30]` = `OPST` set to `ES16` (01),
+two status bits `sentropy[31:30]` = `OPST` set to `ES16` (01),
 indicating successful polling, or with *no* entropy and one of three
 polling failure statuses `BIST` (00), `WAIT` (10), or `DEAD` (11),
 discussed below.
 
 [IMPORTANT]
 ====
-The sixteen bits of randomness in `mentropy[15:0]`=`seed` polled with
-`ES16` status *must be cryptographically conditioned* before they can be
-used as (up to 8 bits of) keying material. We suggest entropy output
-be post-processed in blocks of at least 256 bits, with 128 bit resulting
-output block.
+The sixteen bits of randomness in `seed` (located in `sentropy[15:0]`)
+polled with `ES16` status *must be cryptographically conditioned*
+before they can be used as SSPs (Sensitive Security Parameters) or
+keying material. We suggest entropy output to be post-processed in
+blocks of at least 256 bits, with a 128-bit resulting output block.
 ====
 
-When `OPST` is not `ES16`, `seed` must be set to 0. An implementation
-may safely set reserved and custom bits to zeros. A polling software
-interface must ignore their contents.
+When `OPST` is not `ES16`, `seed` should be set to 0.
```

Comment:
What's the difference between _should_ and _must_? Different standards bodies have different conventions and I don't know what RISC-V's is. To me, "should" is "ought, but might not" and "must" means "this is a bug if it doesn't happen".

Reply:
Yes, it should be a must. Oerr.. it must be a must.. for it to be explicitly testable I guess.

Btw it's very tempting to put error codes there to accompany DEAD and BIST (my implementation did this), but I guess that's an unnecessary information oracle that we're trying to avoid.

Cheers,
- markku